example/radar-custom-color.md

```diff
@@ -0,0 +1,37 @@
+# Radar Chart Custom Color
+
+A custom color accessor function can be passed to the `fill` property of [shapeConfig](http://d3plus.org/docs/#Shape.fill) in order to set specific colors of the [Radar](http://d3plus.org/docs/#Radar). 
```

Comment:
A custom accessor function can be passed to the `fill` property of [shapeConfig](http://d3plus.org/docs/#Shape.fill) in order to set specific colors of a [Radar](http://d3plus.org/docs/#Radar) chart.